packages/client/src/client.js

```diff
@@ -54,10 +55,11 @@ export class PercyClient {
     clientInfo,
     environmentInfo,
     config,
+    labels,
     // versioned api url
     apiUrl = PERCY_CLIENT_API_URL
   } = {}) {
-    Object.assign(this, { token, config: config || {}, apiUrl });
+    Object.assign(this, { token, config: config || {}, apiUrl, labels: labels });
```

Comment:
```suggestion
    Object.assign(this, { token, config: config || {}, apiUrl, buildLabels: labels });
```

Reply:
As the usage is slightly confusing now - one using this.labels and other using labels